survey/questions/translation_first_draft.md

```diff
@@ -89,7 +101,7 @@ Professor
 
 Operational activities
```

Comment:
I'm actually not sure what you mean by "operational activities" ! Could you clarify, here ?

Reply:
This is definitely a language problem :P Here it is meant people working as lab techs, software and hardware developers... (so people not taking care of paper work, but also not pursuing a degree/academic track)